docs/book/features/vsphere_csi_vmc.md

```diff
@@ -0,0 +1,35 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+
+# vSphere CSI Driver - VMware Cloud on AWS (VMC) support
+
+- [Introduction](#introduction)
+- [Deploy vSphere CSI driver on VMC](#deploy-csi-on-vmc)
+
+**Note:** Feature to support vSphere CSI driver on VMC is released with [v2.2.0](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.2.0). Currently, it only support block volume. The supported vSphere version is 6.7U3 and above.
```

Comment:
Can you change this to 
Note: Feature to support vSphere CSI driver on VMC will be available in the vSphere CSI Driver v2.2.0 release. Currently, it supports only block volume. The supported vSphere version is 6.7U3 and above.

Reply:
Are you sure it works on 67u3?

Reply:
Can you confirm which VMC on AWS release we support?
https://docs.vmware.com/en/VMware-Cloud-on-AWS/0/rn/vmc-on-aws-relnotes.html